IdentityCore/src/oauth2/MSIDRequestParameters.m

```diff
@@ -54,4 +54,14 @@ - (NSString *)resource
     return [_target scopeSet];
 }
 
+// Get additional scopes from request params that may not be returned from server
+- (NSOrderedSet<NSString *> *)additionalScopes
+{
+    //.default scope for V1 app will not be returned from server
+    if (self.scopes.count == 1 && [self.scopes.firstObject.lowercaseString hasSuffix:@".default"]){
```

Comment:
nit: since self.scopes creates scope set every time, can we optimize it by using a variable?

Reply:
Sure :)